.github/ISSUE_TEMPLATE/c-feature-request.yml

```
@@ -0,0 +1,69 @@
+name: Submit a feature request
+description: We value your input! If you have any ideas or suggestions for new features, we encourage you to share them here.
+title: "[Feature Request]: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting a feature request! To help us better understand your request, kindly take a moment to provide the following information. Your input is greatly appreciated.
+
+        Description of the feature request:
+        Use case or problem it solves:
+        Any specific requirements or considerations:
+        Any additional information or context:
```

This makes more sense formatted as a bulleted list:
```suggestion
        - Description of the feature request.
        - Use case or problem it solves.
        - Any specific requirements or considerations.
        - Any additional information or context.
```